tests/unit/components/infinity-loader-test.js

```diff
@@ -88,7 +89,7 @@ test('it uses the provided scrollable element', function(assert) {
   assert.equal(scrollable[0], $("#content")[0]);
 });
 
-test('it throws error when scrollable element is not found', function(assert) {
+skip('it throws error when scrollable element is not found', function(assert) {
   assert.expect(1);
 
   this.subject({scrollable: "#nonexistent"});
```

Comment:
Technically this error throws in didInsertElement.

Could we try something more specific to that method?

```
try {
  this.didInsertElement();
} catch
```
or something like that?